docs/index.asciidoc

```diff
@@ -132,6 +132,36 @@ You can configure this plugin to authenticate requests using HTTP Basic authenti
 NOTE: Basic Authentication is not a substitute for SSL, as it provides neither secrecy nor security on its own.
       When used with SSL disabled, HTTP Basic credentials are transmitted in effectively clear-text and can be easily recovered by an adversary.
 
+[id="plugins-{type}s-{plugin}-es-output-notes"]
+==== Using {esf-name} with the Elasticsearch output
+Here are some tips for configuring the {esf} input to work with the elasticsearch output:
+
+* Set the `document_id` in the output configuration when you use the {esf}  input with an {logstash-ref}/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch[Elasticsearch output plugin].
++
+[source,ruby]
+----
+output {
+  elasticsearch {
+      ...
+      document_id => "%{[@metadata][_id]}"
+      ...
+  }
+}
+----
+
+* For {esf-name} v1.9.0 and earlier, rename the field `_id` to `@metadata._id` with a filter:
++
+[source,ruby]
+----
+filter {
+    # support ESF < 1.10
+    if [_id] and ![@metadata][_id] {
+      mutate { rename => { "_id" => "[@metadata][_id]" } }
+    }
+}
+----
+* Starting from version 1.10.0 of {esf-name}, the `_id` field is no longer available, and instead, Logstash now receives the `@metadata._id` field. Configuring `document_id` as shown in the example above is sufficient.
```

Comment:
nit: 
```
Configuring `document_id` as shown in the example above is sufficient.
```

the example just above is the filter, maybe?
```
Configuring `document_id` as shown in the first code example is sufficient.
```

Reply:
or we flip `1.10` and `<=1.9`

@karenzone what do you think?

Reply:
@aspacca, either approach works so there's no wrong answer here.  But given a choice, I prefer  to " flip 1.10 and <=1.9".  We always encourage users to use the latest versions, and we should make that info higher in the doc and easier to find rather than making them keep reading to find relevant info.

Reply:
@karenzone , please, have a look to the changes. thanks :)